dapp_runner/metaclasses.py

```diff
@@ -0,0 +1,17 @@
+"""Metaclasses module."""
```

Comment:
I'd change the name to `singleton` as I'm unsure whether we'll have any more metaclasses ... and the point here is not to have a container for "metaclasses" but rather, if anything, a container for the "singleton" class...